Panic: integer divide by zero when setting a style

```console
panic: runtime error: integer divide by zero

goroutine 1 [running]:
github.com/jedib0t/go-pretty/v6/text.RepeatAndTrim(0x0, 0x0, 0x5, 0x0, 0x0)
	/home/user/go/pkg/mod/github.com/jedib0t/go-pretty/v6@v6.2.1/text/string.go:110 +0x12d
github.com/jedib0t/go-pretty/v6/table.(*Table).initForRenderRowSeparator(0xc00018f5b0)
	/home/user/go/pkg/mod/github.com/jedib0t/go-pretty/v6@v6.2.1/table/table.go:746 +0x1a5
github.com/jedib0t/go-pretty/v6/table.(*Table).initForRender(0xc00018f5b0)
	/home/user/go/pkg/mod/github.com/jedib0t/go-pretty/v6@v6.2.1/table/table.go:569 +0x65
github.com/jedib0t/go-pretty/v6/table.(*Table).Render(0xc00018f5b0, 0xc000128640, 0x4)
	/home/user/go/pkg/mod/github.com/jedib0t/go-pretty/v6@v6.2.1/table/render.go:22 +0x45
main.main.func2(0xc000039140, 0x0, 0x0)
```

My code:

```go
t := table.NewWriter()
style := table.Style{
	Name:    "my-style",
}
t.SetStyle(style)
t.SetOutputMirror(os.Stdout)
t.AppendHeader(table.Row{"#", "Name", "Address", "Phone", "Birthday"})

for i, info := range data.Items {
	t.AppendRows([]table.Row{{i, info.Name, info.Address, info.Phone, info.Birthday}})
}
t.AppendSeparator()
t.AppendFooter(table.Row{"Total", len(data.Items), "", ""})
t.Render()
```

--- a/text/string.go
+++ b/text/string.go
@@ -97,12 +97,13 @@ func Pad(str string, maxLen int, paddingChar rune) string {
 
 // RepeatAndTrim repeats the given string until it is as long as maxRunes.
 // For ex.:
+//  RepeatAndTrim("", 5) == ""
 //  RepeatAndTrim("Ghost", 0) == ""
 //  RepeatAndTrim("Ghost", 5) == "Ghost"
 //  RepeatAndTrim("Ghost", 7) == "GhostGh"
 //  RepeatAndTrim("Ghost", 10) == "GhostGhost"
 func RepeatAndTrim(str string, maxRunes int) string {
-	if maxRunes == 0 {
+	if str == "" || maxRunes == 0 {
 		return ""
 	} else if maxRunes == utf8.RuneCountInString(str) {
 		return str

--- a/text/string_test.go
+++ b/text/string_test.go
@@ -105,13 +105,15 @@ func TestPad(t *testing.T) {
 }
 
 func ExampleRepeatAndTrim() {
+	fmt.Printf("RepeatAndTrim(\"\", 5): %#v\n", RepeatAndTrim("", 5))
 	fmt.Printf("RepeatAndTrim(\"Ghost\", 0): %#v\n", RepeatAndTrim("Ghost", 0))
 	fmt.Printf("RepeatAndTrim(\"Ghost\", 3): %#v\n", RepeatAndTrim("Ghost", 3))
 	fmt.Printf("RepeatAndTrim(\"Ghost\", 5): %#v\n", RepeatAndTrim("Ghost", 5))
 	fmt.Printf("RepeatAndTrim(\"Ghost\", 7): %#v\n", RepeatAndTrim("Ghost", 7))
 	fmt.Printf("RepeatAndTrim(\"Ghost\", 10): %#v\n", RepeatAndTrim("Ghost", 10))
 
-	// Output: RepeatAndTrim("Ghost", 0): ""
+	// Output: RepeatAndTrim("", 5): ""
+	// RepeatAndTrim("Ghost", 0): ""
 	// RepeatAndTrim("Ghost", 3): "Gho"
 	// RepeatAndTrim("Ghost", 5): "Ghost"
 	// RepeatAndTrim("Ghost", 7): "GhostGh"
@@ -119,6 +121,7 @@ func ExampleRepeatAndTrim() {
 }
 
 func TestRepeatAndTrim(t *testing.T) {
+	assert.Equal(t, "", RepeatAndTrim("", 5))
 	assert.Equal(t, "", RepeatAndTrim("Ghost", 0))
 	assert.Equal(t, "Gho", RepeatAndTrim("Ghost", 3))
 	assert.Equal(t, "Ghost", RepeatAndTrim("Ghost", 5))

